README.md

```diff
@@ -112,6 +112,7 @@ Please follow [this](https://github.com/Krypton-Suite/Standard-Toolkit/blob/alph
 =======
 
 ## 2021-08-01 Build 2108 - August 2021 (Canary)
+* Removed `KryptonStatusStrip` & `KryptonToolStrip` - to be implemented in the `Extended Toolkit`
```

Comment:
Not sure that should be the case, as `ToolStrip` and `Status strip`'s are part of the standard `Winform ToolBox`.